strawberryfields/utils.py

```diff
@@ -78,6 +78,7 @@
 quantum states and operations.
 
 .. autosummary::
+   randn
```

Comment:
this will show a function imported from numpy? Don't think it's normal practice to do that

Reply:
It is probably not worth listing it since it is a numpy function, not a an sf function.